README.md

```diff
@@ -347,14 +360,23 @@ Things to keep in mind when running your jobs remotely:
 
 ## Debugging workflow
 
-Here are some tips for fixing unexpected issues occurring remotely.
+Here are some tips for fixing unexpected issues.
 
-[Coming soon]
+### Operation disallowed within distribution strategy scope
+
+**Error like**: Creating a generator within a strategy scope is disallowed, because there is ambiguity on how to replicate a generator (e.g. should it be copied so that each replica gets the same random numbers, or 'split' so that each replica gets different random numbers).
+
+**Solution**: Passing `distribution_strategy='auto'` to `run` API wraps all of your script in a TF distribution strategy based on the cluster configuration provided. You will see the above error or something similar to it, if for some reason an operation is not allowed inside distribution strategy scope. To fix the error, please pass `None` to the `distribution_strategy` param and create a strategy instance as part of your training code as shown in [this](https://github.com/tensorflow/cloud/blob/master/tests/testdata/save_and_load.py) example.
+
+### Version not supported for TPU training
+
+**Error like**: RuntimeError: There was an error submitting the job.Field: tpu_tf_version Error: The specified runtime version '2.3' is not supported for TPU training.  Please specify a different runtime version. See https://cloud.google.com/ml-engine/docs/runtime-version-list for a list of supported versions
+
+**Solution**: Please use TF version 2.1. See TPU Strategy in [Cluster and distribution strategy configuration section](#cluster-and-distribution-strategy-configuration).
```

Comment:
2.3 doesn't work? 2.2 doesn't work? :(

Reply:
Unfortunately no. They are in the process of updating.